keyboards/crkbd/keymaps/waffle/oled.c

```diff
@@ -0,0 +1,182 @@
+#pragma once
+
+extern uint8_t is_master;
```

Comment:
Oh, heads up, this may/will be deprecated in the near future, for the "common" revision.

Reply:
[Split Common crkbd](https://github.com/qmk/qmk_firmware/blob/master/keyboards/crkbd/rev1/common/common.h#L24) 
Not sure if this is relevant but it looks like this split common version uses it? Or are you saying this too will be deprecated in the future? If so, what would you recommend changing about this then?

Reply:
Foostan has a PR that changes/removes it. So ...  Worst case, hide it behind an ifdef, like the oled stuff has.

Reply:
So something like 
`#ifdef SPLIT_KEYBOARD
 #    undef extern uint8_t is_master;
#endif` 
Or am I going down the wrong path here?